molecule/default/molecule-no-systemd.yml

```diff
@@ -24,8 +24,8 @@ platforms:
   - name: debian9
```

Comment:
The comment directly after `platforms:` should be modified to agree with that was written in the role metadata.

Reply:
Please see [bf00fc9](https://github.com/cisagov/ansible-role-cyhy-core/pull/65/commits/bf00fc97d9269709f68d2237486306bc8eb2d202).